cmd/limactl/copy.go

```diff
@@ -49,13 +56,20 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	arg0, err := exec.LookPath("scp")
-	if err != nil {
-		return err
+	defaultTool := Rsync
+	arg0, err := exec.LookPath(string(defaultTool))
+	if err != nil || !strings.HasSuffix(arg0, "rsync") {
+		defaultTool = Scp
+		arg0, err = exec.LookPath(string(defaultTool))
+		if err != nil {
+			return err
+		}
 	}
+	logrus.Infof("using copy tool %q", arg0)
+
 	instances := make(map[string]*store.Instance)
-	scpFlags := []string{}
-	scpArgs := []string{}
+	copyToolFlags := []string{}
+	copyToolArgs := []string{}
```

Comment:
This cannot work since rsync and scp have different flags and arguments. What we need is to add functions to create the copy command, one for rsync and one for scp. Something like:

var command *exec.Cmd
switch tool {
case Rsync:
   command = rsyncCommand(...)
case Scp:
   command = scpCommand(...)
default:
    panic(...)
}

// Run the command...